### DISCUSSION TITLE
Docs alpha: Firestore lite as common use cases too

### DISCUSSION THREAD

**Discussion by peterpeterparker** [firebase/firebase-js-sdk#4376] [General]
I  am checking out the [common use cases](https://modularfirebase.web.app/common-use-cases/firestore/) section of the documentation and, it is really well made.

As there isn't much written about it yet, I think it would be neat to have examples of the new "Firestore Lite" package in that form too.

Typically, "get one document", "get all documents" and "crud"

**Comment by davideast** [comment]
Hey @peterpeterparker! Thank you for the kind words on the docs! It was a labor of love and a lot of fun to build.

We are working on lite examples and will have some soon. Hang tight!

**Reply by peterpeterparker** [reply]
Awesome!

The "lite" version is the one I (will) use in https://github.com/deckgo/deckdeckgo/ and, "how to query a single doc with it" was one of my first question. Looking forward to the sweet examples.
